abstract.tex

```diff
@@ -1,18 +1,12 @@
 \begin{abstract}
 
-The Vera C. Rubin Observatory will advance nearly every area of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
-The LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
-to generate science-ready data products -- processed images, catalogs and alerts. 
+The Vera C.\ Rubin Observatory will advance nearly every area of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST) \cite{2019ApJ...873..111I}.
```

Comment:
"nearly every area of astronomy" seems rather strong.

Reply:
nearly every area-> many areas